sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1254,23 +1253,15 @@ export class CertificateClient {
   }
 
   /**
+   * @internal
+   * @ignore
    * Cancels a certificate creation operation that is already in progress. This operation requires the certificates/update permission.
-   *
-   * Example usage:
-   * ```ts
-   * const client = new CertificateClient(url, credentials);
-   * await client.createCertificate("MyCertificate", {
-   *   issuerName: "Self",
-   *   subject: "cn=MyCert"
-   * });
-   * await client.cancelCertificateOperation("MyCertificate");
-   * ```
    * @summary Cancels a certificate's operation
    * @param certificateName The name of the certificate
    * @param cancel Whether to cancel the operation or not
    * @param {CancelCertificateOperationOptions} [options] The optional parameters
    */
-  public async cancelCertificateOperation(
+  private async cancelCertificateOperation(
```

Comment:
If this is private now, what is it used in?

Reply:
it's being used by the cancel() method on the create and the operation poller. The only difference in both pollers is that the create one _creates_ the certificate, and the operation one just keeps track of the operation. Create also has a KeyVaultCertificate as the return type, while operation has the operation itself as the return type. In order to cancel both, they use the same method: cancelCertificateOperation.